RxSwift/Observables/Observable+Single.swift

```diff
@@ -84,7 +84,7 @@ extension ObservableType {
     - returns: The source sequence with the side-effecting behavior applied.
     */
     // @warn_unused_result(message:"http://git.io/rxs.uo")
-    public func doOn(eventHandler: (Event<E>) throws -> Void)
+    public func doOn(_ eventHandler: (Event<E>) throws -> Void)
```

Comment:
@kzaher I've modernized the `doOn` shortcut syntax as for the conversation (`doOn(next:)` etc.). I don't know about doing the same with `subscribeOn(next:)`, because there is also `subscribeOn` for schedulers and it might get confusing..